data/json/furniture_and_terrain/furniture.json

```diff
@@ -0,0 +1,4917 @@
+[
```

Comment:
I think you may have wrongly comitted this file?

Reply:
Yeah I was just noticing that and was going to delete it.  That won't cause problems will it?

Reply:
I think it won't. These furnitures are already defined in the other files.

Reply:
Done.  Thanks!